csun-metalab/nomi
# NAM-558 -- UX: Change name in feedback form from firstname to Prof. Lastname

# Description
Change name in feedback form from firstname to Prof. Lastname

# Testing
1. Login
2. Tap the Feedback Button
3. Observe the changes.

# Installation
None
